Source/WebKit/UIProcess/Inspector/win/WebInspectorUIProxyWin.cpp

```diff
@@ -405,9 +408,44 @@ void WebInspectorUIProxy::platformShowCertificate(const WebCore::CertificateInfo
     notImplemented();
 }
 
-void WebInspectorUIProxy::platformSave(Vector<WebCore::InspectorFrontendClient::SaveData>&&, bool /* forceSaveAs */)
+void WebInspectorUIProxy::platformSave(Vector<WebCore::InspectorFrontendClient::SaveData>&& saveDatas, bool forceSaveAs)
```

Comment:
Can you merge some part of this function with above code? Mac port has `WebInspectorUIProxy::showSavePanel`.

Reply:
I added a new function `WebInspectorUIProxy::showSavePanelForSingleFile` and merged processes.